src/jsonrpc.js

```diff
@@ -1,12 +1,28 @@
+import bluebird from 'bluebird';
 import logger from './logger';
 
+/**
+ * This is promisify function that convert function with callback to promise
+ * @param {*} func 
+ */
+export const toPromise = func => (...args) =>
+  new Promise((resolve, reject) =>
+    func(...args, (error, result) => (error ? reject(new Error(error.message)) : resolve(result)))
+  );
+
+/**
+ * @param Array array 
+ * @param {*} element 
+ */
+export const isInArray = (array, element) => array.indexOf(element) >= 0;
+
 export class JsonRpc {
   /**
    * Initialize class local variables
-   * @param {*} addresses 
-   * @param {*} fromBlock 
-   * @param {*} toBlock 
-   * @param {*} callback 
+   * @param Array addresses 
+   * @param int fromBlock 
+   * @param int toBlock 
+   * @param function callback 
    */
   constructor(addresses, fromBlock, toBlock, web3, callback = null) {
```

Comment:
I would suggest that we put a warning if there was no callback like your using this function without call back